cloud-common/src/main/java/com/sequenceiq/cloudbreak/service/security/AuthUserService.java

```diff
@@ -0,0 +1,69 @@
+package com.sequenceiq.cloudbreak.service.security;
+
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+import org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationDetails;
+import org.springframework.stereotype.Component;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto;
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+import com.sequenceiq.cloudbreak.auth.altus.GrpcUmsClient;
+import com.sequenceiq.cloudbreak.client.CaasClient;
+import com.sequenceiq.cloudbreak.client.CaasUser;
+import com.sequenceiq.cloudbreak.common.user.CloudbreakUser;
+import com.sequenceiq.cloudbreak.logger.LoggerContextKey;
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.cloudbreak.security.authentication.AuthenticatedUserService;
+import com.sequenceiq.cloudbreak.service.CloudbreakException;
+
+@Component
+public class AuthUserService {
```

Comment:
This service is no longer needed. Please delete it. Check `com.sequenceiq.cloudbreak.security.authentication.AuthUserService` class for more insight.